### PR TITLE
GMWorld example rollup spacecore

### PR DESCRIPTION
**Pull request by chappie1998**

### CODE REVIEW

File: components/gmworld.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+func NewGmworldComponent(root string, home string, node string) *GmworldComponent {
+	return &GmworldComponent{
+		Root:      root,
+		ConfigDir: home,
+	}
+}
+
+func (c *GmworldComponent) InitializeConfig() error {
+	// lightNodePath := filepath.Join(os.Getenv("HOME"), c.ConfigDir+"/"+c.NodeType+"-node")
+	// mkdir -p ~/.vimana/celestia/light-node
```
Comment (mmchougule) [inline components/gmworld.go:21]: please remove this comment

File: config.toml [hunk 2]
```diff
@@ -10,12 +10,24 @@ download = "/tmp/vimana/celestia/init.sh"
...
 [components.avail.light]
 binary = "avail-light"
 download = "/tmp/vimana/avail/init.sh"
 
+
+[components.gmworld]
+
+[components.gmworld.da]
+binary = "gmworld-da"
+download = "/home/ankit/vimana/scripts/rollup_init.sh"
```
Comment (mmchougule) [inline config.toml:25]: let's move these to `~/.vimana/gmd/scripts/rollup_init.sh` so we can also move other scripts to their appropriate directories.

File: config.toml [hunk 3]
```diff
@@ -10,12 +10,24 @@ download = "/tmp/vimana/celestia/init.sh"
...
+
+[components.gmworld.rollup]
+binary = "gmworld-rollup"
+download = "/home/ankit/vimana/scripts/rollup_mocha.sh"
+
```
Comment (mmchougule) [inline config.toml:30]: same as above.

Comment (chappie1998) [reply config.toml:30]: done

File: config.toml (hunk 2 above)
Comment (chappie1998) [reply config.toml]: done

File: components/gmworld.go (hunk 1 above)
Comment (chappie1998) [reply components/gmworld.go]: done

**Review by mmchougule — APPROVED** [review]
lgtm

**Pull request merged**